core/controllers/reader.py

```diff
@@ -300,7 +300,127 @@ def get(self, exploration_id):
 
 
 class StorePlaythroughHandler(base.BaseHandler):
-    """Handles a useful playthrough coming in from the frontend to store it."""
+    """Handles a useful playthrough coming in from the frontend to store it. If
+    the playthrough already exists, it is updated in the datastore.
+    """
+
+    def __init__(self, request, response):
+        """Method that initializes member variables for the handler."""
+        super(StorePlaythroughHandler, self).__init__(request, response)
+        self.exp_issues = None
+        self.issue_schema_version = None
+
+    def _is_issue_in_exp_issues_model(
```

Comment:
nit: define helper methods after the methods which use them.

Reply:
@brianrodri Really, I've had this practice of defining self sufficient methods at the top, and defining methods that use them after this. I've followed that practice in stats_services too

Reply:
Mk, that's fine w/ me, it's not a big deal. Resolved.